src/data/sql-quiz.ts

```diff
@@ -554,6 +554,16 @@ const sqlQuiz = [
       "Database data can either be linked by one-to-one, one-to-many, or many-to-many relationships.",
     Link: "https://www.freecodecamp.org/news/relational-vs-nonrelational-databases-difference-between-sql-db-and-nosql-db/",
   },
+  {
+    Question:
+      "What DDL commands stands for?",
+    Answer: "Data Definition Language",
+    Distractor1:  "Data distinct Language",
+    Distractor2:  "Data Decoded Language",
+    Distractor3:  "Data Developer Language",
+    Explanation:
+      "Data Definition Language includes the set of commands that you use to perform various task related to data definition. You use DDL command to CREATE, ALTER, DROP, TRUNCATE funtions.",
+  },
```

Comment:
You are missing a link here.
All questions must have a link field

Reply:
ok, so should i add the link here now?

Reply:
Yes, you will need to add in the link. You can push up a new commit with that change.
For the link field, please choose either a freeCodeCamp resource, official documentation or a wikipedia article